mlrun/package/packagers/python_standard_library_packagers.py

```diff
@@ -628,9 +629,33 @@ def unpack_path(
 # ----------------------------------------------------------------------------------------------------------------------
 
 # ----------------------------------------------------------------------------------------------------------------------
-# TODO: datetime packagers:
+# datetime packagers:
+# TODO: datetime.date packager.
+# TODO: datetime.time packager.
 # ----------------------------------------------------------------------------------------------------------------------
 
+
+class DatetimePackager(DefaultPackager):
+    """
+    ``datetime.datetime`` packager.
+    """
+
+    PACKABLE_OBJECT_TYPE = datetime.datetime
+    DEFAULT_PACKING_ARTIFACT_TYPE = ArtifactType.RESULT
+    DEFAULT_UNPACKING_ARTIFACT_TYPE = ArtifactType.RESULT
```

Comment:
You can't unpack a result as it is not a `DataItem`. You can simply delete this line.